shared-module/displayio/TileGrid.c

```diff
@@ -17,6 +17,7 @@ void common_hal_displayio_tilegrid_construct(displayio_tilegrid_t *self, mp_obj_
     mp_obj_t pixel_shader, uint16_t width, uint16_t height,
     uint16_t tile_width, uint16_t tile_height, uint16_t x, uint16_t y, uint8_t default_tile) {
     uint32_t total_tiles = width * height;
+    self->inverts = (bool *)m_malloc(total_tiles);
```

Comment:
This will allocate one byte per tile. It may save some code size to do so but it'll leave 7 unused bits.